tests/integration/pydecred/test_rpc.py

```diff
@@ -36,6 +36,28 @@ def test_rpc(config):
     blockchainInfo = rpcClient.getBlockchainInfo()
     assert isinstance(blockchainInfo, rpc.GetBlockChainInfoResult)
 
+    searchRawTransactions = rpcClient.searchRawTransactions(
+        "Dcur2mcGjmENx4DhNqDctW5wJCVyT3Qeqkx"
+    )
+    assert isinstance(searchRawTransactions[0], rpc.SearchRawTransactionsResult)
```

Comment:
Maybe document here that this will fail if tester does not have an `--addrindex`.